### ISSUE TITLE
Opencv error during z-scoring

### ISSUE THREAD

**Issue by Ejdrup** [jbohnslav/deepethogram#44]
OS: Ubuntu 20.10 (Pop_OS!)

I'm using the GUI, as I can't find the documentation to try video copying etc. using command lines.

During copying and z-scoring for videos I get the following error:

```
`[2021-03-18 12:39:05,955][deepethogram.gui.main][INFO] - Copying /home/aske/Documents/Deepethogram/videos/GiDREADD_VTA_coke/Tracking_video2020-03-06T11_24_48_RightHand__cut.mp4 to your DEG directory
[2021-03-18 12:39:05,986][deepethogram.zscore][INFO] - zscoring file: /home/aske/Documents/Deepethogram/test_v1_deepethogram/DATA/Tracking_video2020-03-06T11_24_48_RightHand__cut/Tracking_video2020-03-06T11_24_48_RightHand__cut.mp4
 99%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████▎| 2215/2229 [01:30<00:00, 24.55it/s]
[2021-03-18 12:40:36,217][deepethogram.gui.main][ERROR] - Error initializing video: OpenCV(4.5.1) /tmp/pip-req-build-7m_g9lbm/opencv/modules/imgproc/src/color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cvtColor'
Traceback (most recent call last):
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/gui/main.py", line 224, in initialize_video
    videofile)
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/projects.py", line 143, in add_video_to_project
    zscore_video(os.path.join(video_directory, basename), project)
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/zscore.py", line 125, in zscore_video
    image = reader[i]
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/file_io.py", line 305, in __getitem__
    return self.read(framenum)
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/file_io.py", line 275, in read
    frame = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
cv2.error: OpenCV(4.5.1) /tmp/pip-req-build-7m_g9lbm/opencv/modules/imgproc/src/color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cvtColor'

Traceback (most recent call last):
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/gui/main.py", line 224, in initialize_video
    videofile)
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/projects.py", line 143, in add_video_to_project
    zscore_video(os.path.join(video_directory, basename), project)
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/zscore.py", line 125, in zscore_video
    image = reader[i]
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/file_io.py", line 305, in __getitem__
    return self.read(framenum)
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/file_io.py", line 275, in read
    frame = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
cv2.error: OpenCV(4.5.1) /tmp/pip-req-build-7m_g9lbm/opencv/modules/imgproc/src/color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cvtColor'
```
`
Anything you've encountered before?

**Edit:**
Additionally, when trying to run setup_data.py to test, I get the following:

```
$ python setup_data.py
Traceback (most recent call last):
  File "setup_data.py", line 364, in <module>
    prepare_thumos14_for_testing(args.datadir)
  File "setup_data.py", line 282, in prepare_thumos14_for_testing
    clean_deg_directory(testing_directory)
  File "setup_data.py", line 198, in clean_deg_directory
    model_subfiles = utils.get_subfiles(os.path.join(testing_directory, 'models'))
  File "/home/aske/anaconda3/envs/deg/lib/python3.7/site-packages/deepethogram/utils.py", line 736, in get_subfiles
    files = os.listdir(root)
FileNotFoundError: [Errno 2] No such file or directory: '/home/aske/Documents/Deepethogram/deepethogram-master/tests/deepethogram_testing_thumos14/models'

```

**Comment by jbohnslav** [comment]
Second question: the test dataset is going to be changed dramatically in the future, so don't worry about that. 

First question: there is an error of some kind in your video file. When you open it, e.g. in VLC, do you see there are any "frame skips" near the end of the video? According to the progress bar, it happens about 99% into the video. Depending on your acquisition software, there might've been some frames corrupted by the termination of acquisition.

**Comment by jbohnslav** [comment]
This should be fixed with commit 3d40438a141cd76df79e5e5e3fa09d9a95e5e16c in the new version.
